modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/copy_folder.jsp

```diff
@@ -27,7 +27,7 @@ dlCopyFolderDisplayContext.setViewAttributes(liferayPortletResponse);
 		module="document_library/js/DDMFolderSelector"
 		props='<%=
 			HashMapBuilder.<String, Object>put(
-				"copyActionURL", dlCopyFolderDisplayContext.getActionURL()
+				"copyActionURL", dlCopyFolderDisplayContext.getCopyToURL()
```

Comment:
Parameter and method names need to be consistent. My suggestion is to use `getCopyDLObjectURL` consistently both as a parameter name and as a method name in all display contexts.